C++/Logger/Logger.cpp

```diff
@@ -7,46 +7,47 @@ namespace Toolbox::Log
 {
 
 Logger::Logger (StringSink & sink)
-: _threshold (Level::TRACE)
+: _default (Level::INFO)
+, _threshold (Level::TRACE)
 , _sink (sink)
 , _stringVisitor ()
 {
   
 }
 
-void Logger::log (const Level & level, const String & message) const
+void Logger::log (const String & message) const
 {
-  logAtLevel (level, message);
+  logIfAboveThreshold (_default, message);
```

Comment:
`log` should be overloaded to take a message or a log level and a message:

`log (message);`
`log (_default, message);`

The `log` overload that does not take a level argument should call the log overload that does and should pass in `_default` as the log level